toolkit/tools/internal/network/network.go

```diff
@@ -71,11 +71,24 @@ func CheckNetworkAccess() (err error, hasNetworkAccess bool) {
 		retryAttempts = 10
 		retryDuration = time.Second
 		squashErrors  = false
+		activeStatus  = "active"
 	)
 
 	err = retry.Run(func() error {
-		err := shell.ExecuteLive(squashErrors, "ping", "-c", "1", "www.microsoft.com")
-		hasNetworkAccess = err == nil
+		err := shell.ExecuteLive(squashErrors, "systemctl", "restart", "systemd-networkd-wait-online")
```

Comment:
Could you add a comment to the function indicating this may only be run in the context of an ISO? Either that or it should fail gracefully if systemd is missing.

Reply:
Fixed.